.github/workflows/long-running-azure.yaml

```diff
@@ -48,6 +51,9 @@ on:
       - ".github/workflows/long-running-azure.yaml"
 
 env:
+  # This is debugging purpose.
+  FORCE_RUN: 'false'
```

Comment:
Would this need to be updated to true and merged into main to take effect? If so, would it make more sense to it configurable by making it a workflow input with default set to false, so that we don't have to change it multiple times to force an update.

Reply:
This is not working since we need to use fork repo. I will enable manual trigger for this workflow to enforce deploy from main branch.